reqwest-tracing/src/reqwest_otel_span_builder.rs

```diff
@@ -64,6 +86,11 @@ pub fn default_on_request_success(span: &Span, response: &Response) {
         span.record(OTEL_STATUS_CODE, span_status);
     }
     span.record(HTTP_RESPONSE_STATUS_CODE, response.status().as_u16());
+    #[cfg(feature = "deprecated_attributes")]
+    {
+        span.record(HTTP_STATUS_CODE, response.status().as_u16());
+        span.record(HTTP_USER_AGENT, user_agent.as_str());
```

Comment:
When I tried to build with `--all-features`, this doesn't work because there is no `user_agent` in scope.

Am I missing something?